includes/admin/views/html-admin-page-status-report.php

```diff
@@ -464,6 +467,10 @@
 					if ( false != $plugin['network_activated'] ) {
 						$network_string = ' &ndash; <strong style="color:black;">' . __( 'Network enabled', 'woocommerce' ) . '</strong>';
 					}
+
+					if ( in_array( $plugin['plugin'], array_keys( $untested_plugins ) ) ) {
```

Comment:
Could change to `array_key_exists()`.

Reply:
Hm, I was going to argue that you can't do that as array_key_exists doesn't support the "strict comparison" argument that in_array() does. Then I realised I hadn't used it anyway ...

I can add the strict flag on - or swap to array_key_exists() - any preference?

Reply:
Are all strings, so no need for strict comparison.

But better move it outside this block, since we are checking it for all plugins and not only `woothemes.com` or `woocommerce.com` plugins.

Reply:
non woothemes / woocommerce plugins won't be in the $untested_plugins array though - so that's OK?

Reply:
Ah, I got you ... fix coming in a second ...